### DISCUSSION TITLE
QR code not scanning (QR code disappears when attempting to scan)

### DISCUSSION THREAD

**Discussion by MsAPotter** [Authenticator-Extension/Authenticator#627] [General]
Upon install and during setup of first QR code scan, QR code appears on the screen, but disappears when I try to hover over and scan. I have removed and re-installed the Chrome Authenticator but the same thing keeps happening.

<img width="1440" alt="Screen Shot 2021-02-09 at 1 45 28 PM" src="https://user-images.githubusercontent.com/47091947/107412100-4867a200-6add-11eb-9f0d-9f77e5b571eb.png">

<img width="1440" alt="Screen Shot 2021-02-09 at 1 45 33 PM" src="https://user-images.githubusercontent.com/47091947/107412122-4f8eb000-6add-11eb-8a78-cab7cec6766b.png">

This is what appear on the the Javascript console:
DevTools failed to load SourceMap: Could not load content for chrome-extension://bhghoamapcdpbohphigoooaddinpkbai/dist/popup.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME
DevTools failed to load SourceMap: Could not load content for chrome-extension://bhghoamapcdpbohphigoooaddinpkbai/dist/argon.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME
DevTools failed to load SourceMap: Could not load content for chrome-extension://bhghoamapcdpbohphigoooaddinpkbai/css/popup.css.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME

**Comment by mymindstorm** [comment]
This is intentional, that QR is just a tutorial image to tell you that you should click and drag over a QR on the page. Could you try scanning this test QR? https://chart.googleapis.com/chart?chs=200x200&chld=M%7C0&cht=qr&chl=otpauth://totp/test?secret=WXKU2NN3EHFQ7I3MIGCLW2ZNN5ZNDYRN

**Comment by MsAPotter** [comment]
Hm ok.
Yeah that worked!

**Comment by MsAPotter** [comment]
I had set it up with my AWS account a couple months ago, but last night/this morning I was running into issues with syncing it to the AWS sign-in. I kept getting the error: "Your multi-factor authentication (MFA) token might be out of sync with our server...."; and I couldn't login.

**Comment by mymindstorm** [comment]
Have you tried going to the menu -> Sync clock?

**Comment by MsAPotter** [comment]
Yeah. It said the sync clock was fine 🤷‍♀️

**Comment by mymindstorm** [comment]
Could you try clicking the qr icon in the top right corner of the entry, add that to a phone, and try logging in? If that fails, you should try restoring from a backup or recovering the account. I was able to successfully login to my AWS account using the extension. Lastly, are you using browser sync?
